tests/support/bower.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "purescript-test-suite-support",
   "dependencies": {
-    "purescript-eff": "0.1.0",
-    "purescript-prelude": "0.1.3",
-    "purescript-assert": "0.1.1",
-    "purescript-st": "0.1.0",
-    "purescript-console": "0.1.0",
-    "purescript-functions": "0.1.0"
+    "purescript-assert": "^1.0.0-rc.1",
+    "purescript-console": "^1.0.0-rc.1",
+    "purescript-eff": "^1.0.0-rc.1",
+    "purescript-functions": "^1.0.0-rc.1",
+    "purescript-prelude": "^1.0.0-rc.1",
+    "purescript-st": "^1.0.0-rc.1"
```

Comment:
I think it might make more sense to use exact versions here; then we don't have to worry about something outside the repository changing and breaking the tests.